src/solvers/postSolver.ml

```diff
@@ -154,13 +154,7 @@ struct
 
   module VH = Hashtbl.Make (S.Var)
   (* starts as Hashtbl for quick lookup *)
-  let starth =
-    (* VH.of_list S.starts *) (* TODO: BatHashtbl.Make.of_list is broken, use after new Batteries release *)
-    let starth = VH.create (List.length S.starts) in
-    List.iter (fun (x, d) ->
-        VH.replace starth x d
-      ) S.starts;
-    starth
```

Comment:
Hmm, testing this isn't very straightforward indeed. Because this only matters when there's a fixpoint verification error (on a start variable!). Unless something else is wrong, this wouldn't even show.